src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/tests/adapters/__snapshots__/TreeTableAdapter.test.js.snap

```diff
@@ -53,7 +53,11 @@ exports[`Render data with pencil button when onItemEdit callback is passed 1`] =
         >
           <div
             class="cellContent"
-          />
+          >
+            <div
+              class="publishIndicator publishIndicator"
```

Comment:
That's odd... If this indicator is not shown in the real application, these DOM elements also don't exist... Can you find out what is wrong here? Maybe the test data you are using is not what is actually returned by the API?